docs/basics/app-anatomy.md

```diff
@@ -44,102 +44,107 @@ In general, the core of the state-machine is defined in a file called `app.go`.
 
 See an example of `BeginBlocker` and `EndBlocker` functions from [`gaia`](https://github.com/cosmos/gaia)
 
-+++ https://github.com/cosmos/gaia/blob/f41a660cdd5bea173139965ade55bd25d1ee3429/app/app.go#L224-L232
++++ https://github.com/cosmos/gaia/blob/d00db033d861c4f59e07038e61bcaf39274ff6da/app/app.go#L232-L240
 
 ### Register Codec
 
-The `MakeCodec` function is the last important function of the `app.go` file. The goal of this function is to instantiate a [codec `cdc`](../core/encoding.md) (e.g. amino) initialize the codec of the SDK and each of the application's modules using the `RegisterLegacyAminoCodec` function.
+The `MakeCodecs` function is the last important function of the `app.go` file. The goal of this function is to instantiate a [codec `cdc`](../core/encoding.md) (e.g. Protobuf), and initialize the codec of the SDK and each of the application's modules using the `RegisterInterfaces` function.
+
+To register the application's modules, the `MakeCodecs` creates a codec which will be passed as argument into the `RegisterInterfaces` method on `ModuleBasics`. `ModuleBasics` is a [basic manager](../building-modules/module-manager.md#basicmanager) which lists all of the application's modules. It is instanciated in the `init()` function, and only serves to easily register non-dependant elements of application's modules (such as codec). To learn more about the basic module manager, click [here](../building-modules/module-manager.md#basicmanager).
+
+Please note that in this version of the SDK, the `MakeCodecs` function actually creates two codecs:
 
-To register the application's modules, the `MakeCodec` function calls `RegisterLegacyAminoCodec` on `ModuleBasics`. `ModuleBasics` is a [basic manager](../building-modules/module-manager.md#basicmanager) which lists all of the application's modules. It is instanciated in the `init()` function, and only serves to easily register non-dependant elements of application's modules (such as codec). To learn more about the basic module manager, click [here](../building-modules/module-manager.md#basicmanager).
+- an `appCodec`, which will be used by default across the SDK, defaults to Protocol Buffers,
+- a `legacyAmino` codec, as some legacy parts of the SDK still uses Amino for backwards-compatibility. This `legacyAmino` codec should not be used by app developers, and will be removed in the next release.
 
-See an example of a `MakeCodec` from [`gaia`](https://github.com/cosmos/gaia):
-    
-+++ https://github.com/cosmos/gaia/blob/f41a660cdd5bea173139965ade55bd25d1ee3429/app/app.go#L64-L70
+See an example of a `MakeCodecs` from [`gaia`](https://github.com/cosmos/gaia):
+
++++ https://github.com/cosmos/gaia/blob/d00db033d861c4f59e07038e61bcaf39274ff6da/app/app.go#L71-L81
```

Comment:
gaia still hasn't been updated to use proto, this snippet is still Amino unfortunately.

Reply:
Referencing simapp code now.